content/en/blog/why-choose-fission.md

```diff
@@ -0,0 +1,105 @@
+date = "2022-03-23T08:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Fission is one of the popular open source Kubernetes serverless framework. In this post we help you understand why should you choose Fission."
+categories = ["Fission"]
+type = "blog"
+images = ["images/featured/why-fission-serverless-framework.png"]
++++
+
+The Serverless paradigm is empowering developers to focus only on building the application and not worry about anything else.
+Kubernetes has become an industry standard for hosting cloud native or container based microservice applications.
+It works seamlessly across public/private clouds and provides a common platform without any vendor lock-in.
+Hence, it makes naturally a good choice to build serverless frameworks on top of Kubernetes.
+
+There are a handful of Kubernetes serverless frameworks out there and Fission is one of the popular ones.
+
+## What is Fission
+
+Fission is the Kubernetes serverless framework that helps you build serverless applications.
+It allows you to **write and deploy functions almost instantly**.
+You don't need to create any containers or deal with Docker registry.
+With just a few commands your application will be live.
+You can host your applications on your laptop, public cloud or in your private data center as it runs on Kubernetes.
+If you're wondering why to choose Fission Kubernetes serverless framework, read on and let us tell you why.
+
+{{< figure src="/images/featured/why-fission-serverless-framework.png" alt="Reasons to choose Fission Kuberenetes Serverless Framework" height="600" width="800">}} Why Choose Fission Kuberenetes Serverless Framework
+
+
+## Why Choose Fission?
+
+### Customizable Environments
+
+Fission provides unmatched extensibility.
+The core of Fission is written in **Go**, it has [Environments](/docs/concepts/#environments) which are the language-specific parts.
+An environment in Fission contains only the adequate code that is required to run a function.
+Fission currently supports **NodeJS, Python, Go, Java, Ruby, PHP**, and **.NET**.
+You can check out all the [Fission supported environments](https://environments.fission.io/).
+
+{{< figure src="https://fission.io/docs/concepts/assets/trigger-function-environment.png" alt="Environment in Fission" height="500" width="500">}} Functions execute inside an environment.
+
+The key of Fission resides in customizable environments.
+As a developer, you can **modify any of Fission's supported environments** and rebuild them.
+Not only that, you can even create a new environment from scratch.
+We have a blog post about using [PostgreSQL with Fission](../blog/how-to-use-postgresql-database-with-fission-functions/) where we show how to modify an environment.
+Even the components in Fission are stored as kubernetes CRDs and are customizable.
+
+Due to customizability of environments, there are many companies that prefer using Fission.
+Some of them are even offering serverless function hosting services.
```

Comment:
Many have of them have build Function as a service(FaaS) platform on top of Fission for enterprise hosting.